app/uk/gov/hmrc/exports/models/declaration/ExportsDeclaration.scala

```diff
@@ -44,6 +45,51 @@ case class ExportsDeclaration(
 )
 
 object ExportsDeclaration {
+  
+  @deprecated("Please use apply with `Choice` class", since = "2019-08-14")
```

Comment:
Cant we just change the existing usages rather than add a deprecated constructor?